engine/apps/slack/scenarios/alertgroup_appearance.py

```diff
@@ -57,7 +57,8 @@ def process_scenario(self, slack_user_identity, slack_team_identity, payload, ac
 
         # This is a special case for amazon sns notifications in str format CHEKED
         if (
-            AlertReceiveChannel.INTEGRATION_AMAZON_SNS is not None
+            hasattr(AlertReceiveChannel, "INTEGRATION_AMAZON_SNS")
+            and AlertReceiveChannel.INTEGRATION_AMAZON_SNS is not None
```

Comment:
Seems like `hasattr(AlertReceiveChannel, "INTEGRATION_AMAZON_SNS")` check should be enough, so checking for `AlertReceiveChannel.INTEGRATION_AMAZON_SNS is not None` is not required?

Reply:
As you said I think that checking is wasted, so I removed this line. Thanks!